charts/karma/templates/deployment.yaml

```diff
@@ -64,6 +64,14 @@ spec:
           {{- end }}
           {{- with .Values.env }}
 {{ toYaml . | indent 10 }}
+          {{- end }}
+          {{- if gt (len .Values.configurations) 0 }}
+          command:
```

Comment:
Is command mandatory when setting args? I have a doubt here!

Reply:
You are right. Tested it without the command and still fine.